tests/cmdline/commands/test_code/test_code_exportyml

```diff
@@ -0,0 +1,8 @@
+append_text: ''
+computer: localhost
+default_calc_job_plugin: core.arithmetic.add
+description: code
+filepath_executable: /bin/cat
+label: code
+prepend_text: ''
+use_double_quotes: 'False'
```

Comment:
Will this be correctly interpreted? Should this not be a literal `false` or at least without quotes? Ideal would be to have a round-trip test and call `verdi code create` with this yaml and see if you get equivalent codes

Reply:
Yes, the `False` as a string is working, I add a round trip test for it.